pkg/apis/kubeflow.org/v2alpha1/trainjob_types.go

```diff
@@ -48,6 +48,43 @@ type TrainJob struct {
 	Status TrainJobStatus `json:"status,omitempty"`
 }
 
+const (
+	// TrainJobSuspended means the TrainJob is suspended.
+	TrainJobSuspended string = "Suspended"
+
+	// TrainJobComplete means that the TrainJob has completed its execution.
+	TrainJobComplete string = "Complete"
```

Comment:
Please can you create an issue in JobSet to rename Completed condition to Complete as we discussed?

Reply:
Yeah, we can